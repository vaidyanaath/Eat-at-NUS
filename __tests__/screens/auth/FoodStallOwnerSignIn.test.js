import React from 'react';
import renderer from 'react-test-renderer';

import FoodStallOwnerSignIn from '../../../src/screens/auth/FoodStallOwnerSignIn';

describe('<FoodStallOwnerSignIn />', () => {
    
    it('Renders correctly', () => {
        const tree = renderer.create(<FoodStallOwnerSignIn />).toJSON();
        expect(tree).toMatchSnapshot();
    });
    
});