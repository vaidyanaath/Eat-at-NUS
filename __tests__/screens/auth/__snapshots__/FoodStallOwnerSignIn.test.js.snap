// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FoodStallOwnerSignIn /> Renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#FFFFFF",
        "flexBasis": 0,
        "flexDirection": "column",
        "flexGrow": 1,
        "flexShrink": 1,
        "height": "100%",
        "paddingBottom": 15,
        "paddingLeft": 15,
        "paddingRight": 15,
        "paddingTop": 30,
        "width": "100%",
      },
      Object {},
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "justifyContent": "center",
          "width": "100%",
        },
        Object {
          "minHeight": "90%",
        },
      ]
    }
  >
    <View
      onLayout={[Function]}
      style={
        Array [
          Object {
            "flex": 1,
          },
          Object {
            "paddingBottom": 0,
          },
        ]
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#2C365A",
                "fontFamily": "SourceSansPro-Regular",
                "fontSize": 22,
              },
              Object {
                "marginVertical": 10,
              },
            ]
          }
        >
          Food Stall Owner
        </Text>
        <TextInput
          autoCorrect={false}
          keyboardType="email-address"
          onChangeText={[Function]}
          placeholder="Email"
          style={
            Object {
              "backgroundColor": "#FFFFFF",
              "borderRadius": 5,
              "borderWidth": 1,
              "height": 45,
              "marginVertical": 5,
              "padding": 10,
              "width": 280,
            }
          }
          value=""
        />
        <TextInput
          autoCorrect={false}
          onChangeText={[Function]}
          placeholder="Password"
          secureTextEntry={true}
          style={
            Object {
              "backgroundColor": "#FFFFFF",
              "borderRadius": 5,
              "borderWidth": 1,
              "height": 45,
              "marginVertical": 5,
              "padding": 10,
              "width": 280,
            }
          }
          value=""
        />
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "alignSelf": "center",
              "backgroundColor": "#2C365A",
              "borderRadius": 10,
              "elevation": 3,
              "height": 48,
              "justifyContent": "center",
              "marginBottom": 5,
              "marginLeft": 5,
              "marginRight": 5,
              "marginTop": 5,
              "opacity": 1,
              "paddingBottom": 10,
              "paddingLeft": 10,
              "paddingRight": 10,
              "paddingTop": 10,
              "shadowcolor": "#000",
              "shadowoffset": "0 1px",
              "shadowopacity": 0.22,
              "shadowradius": 2.22,
              "width": 280,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#2C365A",
                  "fontFamily": "SourceSansPro-Regular",
                  "fontSize": 22,
                },
                Object {
                  "color": "#FFFFFF",
                  "fontSize": 17,
                },
              ]
            }
          >
            Sign In
          </Text>
        </View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "alignSelf": "center",
              "backgroundColor": "#FFFFFF",
              "borderRadius": 10,
              "justifyContent": "center",
              "marginBottom": 5,
              "marginLeft": 5,
              "marginRight": 5,
              "marginTop": 0,
              "opacity": 1,
              "paddingBottom": 10,
              "paddingLeft": 10,
              "paddingRight": 10,
              "paddingTop": 10,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#2C365A",
                  "fontFamily": "SourceSansPro-Regular",
                  "fontSize": 15,
                },
                Object {
                  "color": "#ff7f50",
                  "fontSize": 17,
                },
              ]
            }
          >
            Forgot Password?
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "justifyContent": "center",
          "width": "100%",
        },
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "center",
          "minHeight": 80,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#2C365A",
            "fontFamily": "SourceSansPro-Regular",
            "fontSize": 15,
          },
          Object {
            "color": "#2C365A",
            "textAlign": "right",
          },
        ]
      }
    >
       Don't have an account? 
    </Text>
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "backgroundColor": "#FFFFFF",
          "borderRadius": 10,
          "justifyContent": "flex-start",
          "marginBottom": 5,
          "marginLeft": 5,
          "marginRight": 5,
          "marginTop": 5,
          "opacity": 1,
          "paddingBottom": 10,
          "paddingLeft": 10,
          "paddingRight": 10,
          "paddingTop": 10,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#2C365A",
              "fontFamily": "SourceSansPro-Regular",
              "fontSize": 15,
            },
            Object {
              "color": "#ff7f50",
            },
          ]
        }
      >
         Sign Up 
      </Text>
    </View>
  </View>
</View>
`;
